src/gui/components/navbar/navbar.styles.js

```diff
@@ -0,0 +1,15 @@
+import { css } from 'glamor'
+import { primaryColor } from '../../styles'
+
+const layout = css({
+  display: 'flex',
+  alignItems: 'center',
+})
+
+export const title = css({
+  fontSize: '1.2em',
+  fontWeight: 'bolder',
+  paddingLeft: '10px',
+})
+
+export default css(primaryColor, layout)
```

Comment:
Yeah, so it make my point, I don't think this is good to export the 'aggregated' version as default, don't you think ?